corehq/apps/hqwebapp/static/hqwebapp/less/_hq/modals.less

```diff
@@ -125,3 +125,84 @@
         transform: translate(0, 0) !important;
     }
 }
+
```

Comment:
less indentation should be 2, not 4

Reply:
this should live in app manager styles

Reply:
add a prefix like `hq-add-new-module-modal`

Reply:
also, where you do add it, please add a comment next to the styles that in b5 we should use as much of the built-in utilities as possible to achieve the same layout

Reply:
Moved in f618066a49c4d8ca4b0e2e3aa2e76de5fa535297
Prefix and comment added in c8b48743dc0e17c033f056af8b2d8bad83bfc735